signinwithapplebutton/src/main/java/com/willowtreeapps/signinwithapplebutton/SignInWithAppleService.kt

```diff
@@ -3,15 +3,31 @@ package com.willowtreeapps.signinwithapplebutton
 import android.net.Uri
 import android.os.Parcel
 import android.os.Parcelable
+import android.util.Pair
 import androidx.fragment.app.FragmentManager
+import com.willowtreeapps.signinwithapplebutton.view.SignInWebViewDialogFragment
 import java.util.*
 
 class SignInWithAppleService(
-    private val clientId: String,
-    private val redirectUri: String,
-    private val scope: String
+    private val fragmentTag: String,
+    private val fragmentManager: FragmentManager,
+    private val configuration: SignInWithAppleConfiguration,
+    private val callback: (SignInWithAppleResult) -> Unit
 ) {
 
+    constructor(
+        fragmentTag: String,
+        fragmentManager: FragmentManager,
```

Comment:
I think I'd like to swap the order of these two arguments. What I see in Android functions is that major things come before minor things, e.g. contexts before resources. Is that intuition right?

Reply:
Or at least, that's the order in Fragment.show().

Reply:
Done